guide/src/reference/attributes/on-js-imports/indexing-getter-setter-deleter.md

```diff
@@ -0,0 +1,39 @@
+# `indexing_getter`, `indexing_setter`, and `indexing_deleter`
+
+These three attributes indicate that a method is a getter, setter, or deleter
+method on the receiver object itself, rather than one of the receiver's
+properties. This is useful for binding to special objects like `Array` and
+`NodeList` that override indexing operations.
```

Comment:
Not sure about this part.
Firstly, normal `getter` and `setter` with `structural` can also access properties pretty much the same way, the main difference is that the name of the property with normal ones is always the same and specified in name of the method or in the value of the property, while in indexing is it dynamic and is specified in the argument. Would it be possible to describe the difference more clearly here?
Secondly, I do not think that are good example. `NodeList` definitely does not generate any of these 3 attributes, not sure about `Array`. I think just not giving an example can be fine, but it is possible to write JS code that corresponds to the Rust example (it would probably use `Proxy`, check out the test: https://github.com/rustwasm/wasm-bindgen/blob/9b65e57585c6ad334ac5fb3867656f3e734747a5/crates/webidl-tests/simple.js#L96-L108). Like this I think:
```js
new Proxy({}, {
    get(obj, prop) {
        return obj.hasOwnProperty(prop) ? obj[prop] : 0xFFFFFFFF;
    },
    set(obj, prop, value) {
        obj[prop] = value;
    },
    deleteProperty(obj, prop) {
        delete obj[prop];
    },
});
```